docs/snippets/react/nextjs-configure-svgr.js.mdx

```diff
@@ -0,0 +1,25 @@
+```js
```

Comment:
Can you adjust this file and the TypeScript variant to align with our formatting pattern?